.github/workflows/release.yaml

```diff
@@ -16,28 +16,29 @@ jobs:
     uses: ./.github/workflows/check.yaml
     secrets: inherit
 
+
   release:
     needs: check
-    runs-on: ${{ matrix.runs-on }}
-    strategy:
-      fail-fast: false
-      matrix:
-        runs-on: [[ubuntu-22.04]]
+    runs-on: ubuntu-24.04
```

Comment:
Oh nice! We'll only need to run this on a single architecture, because it's simply uploading the pre-built charms. :+1: This will make the charm_release template in automation simpler, because we can remove the runs_on variable. :ok_hand:

Reply:
Yes, that's one of the major benefits (others major benefits are: reduce the resource uses, reduce the time for releasing workflow)